.circleci/config.yml

```diff
@@ -252,8 +270,8 @@ jobs:
             ./mvnw -B org.jacoco:jacoco-maven-plugin:report org.jacoco:jacoco-maven-plugin:report-aggregate clean install -Pnon-confidential-tests,coverage -Djavax.net.ssl.trustStore=../LocalTrustStore -Djavax.net.ssl.trustStorePassword=changeit  -DskipSignatureCheck=$SKIP_SIGNATURE_CHECK -ntp
             # ./mvnw -B org.jacoco:jacoco-maven-plugin:report org.jacoco:jacoco-maven-plugin:report-aggregate clean install -Pnon-confidential-tests,coverage -Djavax.net.ssl.trustStore=../LocalTrustStore -Djavax.net.ssl.trustStorePassword=changeit -ntp
       - run:
-          name: send coverage
-          command: bash <(curl -s https://codecov.io/bash) -F ${TESTING_PROFILE//-} || echo "Codecov did not collect coverage reports"
+          name: send coverage for non-confidential integration tests
+          command: bash <(curl -s https://codecov.io/bash) -F unit-tests_and_non-confidential-tests || echo "Codecov did not collect coverage reports"
       - run:
           name: check that JPA classes are consistent with migrations
           command: |
```

Comment:
cannot comment on it directly, why is `circle_ci_test_data.zip.enc` changing?
Does this have to do with my PR to reissue the gitlab tokens? 
However, I don't see my commit in the list of commits (has this been updated from develop?)

Reply:
Yes I was looking at that too. I think it's because I updated that file from develop (to get my tests to work) but the changes section on the PR compares my code with the most recent common commit.

If you do a two dot difference check `circle_ci_test_data.zip.enc` doesn't show up, https://github.com/dockstore/dockstore/compare/develop..feature/flynn/4268/seperate_bitbucket_tests